addons/l10n_it_edi/views/l10n_it_view.xml

```diff
@@ -83,6 +83,30 @@
         </field>
     </record>
 
+    <record id="view_out_invoice_tree_inherit" model="ir.ui.view">
+        <field name="name">account.move.tree.inherit</field>
+        <field name="model">account.move</field>
+        <field name="inherit_id" ref="account_edi.view_out_invoice_tree_inherit" />
+        <field name="arch" type="xml">
+            <field name="state" position="before">
+                <field name="l10n_it_edi_transaction" optional="hide"/>
+                <field name="l10n_it_edi_attachment_id" optional="hide"/>
+            </field>
+        </field>
+    </record>
+
+    <record id="view_account_invoice_filter" model="ir.ui.view">
+        <field name="name">account.invoice.select.inherit</field>
+        <field name="model">account.move</field>
+        <field name="inherit_id" ref="account_edi.view_account_invoice_filter"/>
+        <field name="arch" type="xml">
+            <xpath expr="//search/field[@name='journal_id']" position="after">
+                <field name="l10n_it_edi_transaction"/>
+                <field name="l10n_it_edi_attachment_id"/>
+            </xpath>
+        </field>
+    </record>
```

Comment:
```suggestion
```
Problem is that all companies see it and I don't know if it is really necessary to be searching for that field.  (and users can add it custom anyways)

Reply:
I don't want to see 100 invoices and look up the id_transaction by hand if I have an error on the IAP server logs.

To me, the whole PR is useless if I cannot search moves by `l10n_it_edi_transaction`, because that's the main tool to actually read the logs. Can't we just put a `groups` directive for debug?

Reply:
If we cannot search here, then I prefer having a separate more technical EDI Documents view

Reply:
I've put the `groups="base.group_no_one"` on the searches.

Reply:
But the user can do a custom search...  (it is not that the user can not do it)